src/api/article/utils.js

```diff
@@ -0,0 +1,103 @@
+import fromPairs from 'lodash/fromPairs';
+import omit from 'lodash/omit';
+
+import { getImageUrl } from 'services/fibery/getters';
+import { parseSoundcloudUrl } from 'services/soundcloud';
```

Comment:
I guess soundcloud also may be moved to `lib` one day?

Reply:
only if we need to share the code to frontend